sdk/tables/Azure.Data.Tables/src/TableClient.cs

```diff
@@ -67,7 +67,7 @@ public virtual TableSasBuilder GetSasBuilder(string rawPermissions, DateTimeOffs
         [ForwardsClientCalls]
         public virtual Response<TableItem> Create(CancellationToken cancellationToken = default)
         {
-            var response = _tableOperations.RestClient.Create(new TableProperties(_table), null, queryOptions: new QueryOptions() { Format = _format }, cancellationToken: cancellationToken);
+            var response = _tableOperations.Create(new TableProperties(_table), null, queryOptions: new QueryOptions() { Format = _format }, cancellationToken: cancellationToken);
```

Comment:
BTW, I was thinking about moving to wrapping RestClients in data plane libraries, because the only additional thing the "public" client add is scopes and their names are wrong anyway.